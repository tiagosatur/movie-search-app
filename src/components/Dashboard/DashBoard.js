import React from 'react';
import { useSelector } from 'react-redux';


const Dashboard = () => {
    const {
        user: {
          user
        },
    } = useSelector(state => state);

    return(
        <div>
        <h2>Dashboard</h2>
        <p>
            Seja bem vindo 
            <strong> { user.username} </strong>!
        </p>
    </div>
    )
}

export default Dashboard;