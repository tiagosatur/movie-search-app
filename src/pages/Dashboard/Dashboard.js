import React from 'react';
import { useSelector } from 'react-redux';


const Dashboard = (props) => {
    
    const {
        user: {
          user: { username },
          isLogged
        },
    } = useSelector(state => state);
    // console.log('DASHBOARD', isLogged);
    // if(!isLogged) return <h4>You must login to see this page...</h4>

    return(
        <div className='dashboard'>
            <h2>Dashboard</h2>
            <p>
                Seja bem vindo 
                <strong> { username } </strong>!
            </p>
        </div>
    )
}

export default Dashboard;