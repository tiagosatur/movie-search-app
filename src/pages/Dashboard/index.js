export { default as Dashboard } from './DashBoard'