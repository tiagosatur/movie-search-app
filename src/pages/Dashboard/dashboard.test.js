import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Dashboard from './Dashboard';

describe('Dashboard tests', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(
     <Dashboard />
    ).dive();
  });
  
  it('should render the Dashboard element', () => {
    expect(wrapper.find('.dashboard').exists()).toBe(true);
  });

  it('should call Dashboard snapshot correctly', () => {
    const tree = renderer
        .create(wrapper)
        .toJSON();
    expect(tree).toMatchSnapshot();
    });
})