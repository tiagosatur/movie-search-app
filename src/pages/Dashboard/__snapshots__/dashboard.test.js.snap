// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard tests should call Dashboard snapshot correctly 1`] = `null`;
